app/workers/deletes.php

```diff
@@ -305,6 +310,21 @@ protected function deleteByGroup(array $filters, Database $database, callable $c
         Console::info("Deleted {$count} document by group in " . ($executionEnd - $executionStart) . " seconds");
     }
 
+    protected function deleteCertificates(Document $document)
+    {
+        $domain = $document->getAttribute('domain');
+        $directory = APP_STORAGE_CERTIFICATES . '/' . $domain;
+        $checkTraversal = realpath($directory) === $directory;
+
+        if($domain && $checkTraversal && is_dir($directory)) {
+            array_map('unlink', glob($directory.'/*.*'));
```

Comment:
This is a bit scary. I would add extra validation to make sure we don't delete any top-level dir or just target the specific file if know their names.

Reply:
I already do that with the $checkTraversal variable :+1:

Reply:
👌🏻